src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+const employeesList = document.querySelector('ul');
```

Comment:
You could create only one variable 'employeesList' and use it like [... employeesList.children] instead of creating another variable 'employees'